app/models/reviews-model.js

```diff
@@ -129,3 +130,28 @@ exports.selectReviews = (query = { sort_by: "created_at" }) => {
       errorMessage: "Bad request, incorrect method",
     });
 };
+
+exports.insertReview = (newReview, next) => {
+  const { owner, title, review_body, designer, category } = newReview;
+  if (owner && title && review_body && designer && category) {
```

Comment:
if any of these fields are `NOT NULL` in the database then `db.query` would have thrown an error for you

handling it youself beforehand is totally valid though!